active_record_host_pool.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 3.0.0"
```

Comment:
```suggestion
  s.required_ruby_version = ">= 3.1.0"
```

Reply:
I wasn't sure if we wanted to drop the gem from bundling with Ruby 3.0 altogether or not but happy to require Ruby 3.1+ 🤷

Reply:
I would prefer to drop it completely, rather than allow an untested version.